transient/editor.py

```diff
@@ -170,14 +170,14 @@ def _mount_root(self) -> None:
             try:
                 self.run_command_in_guest(
                     [
-                        f"mount -t {fstype} {path} /mnt > /dev/null",
+                        f"mount -t {fstype} {path} /mnt &> /dev/null",
```

Comment:
Doing this would make it very difficult for users to detect why a mount they are expecting to work is not. Instead, I think you can add `capture_stderr=True` to this call. That way, the error will be visible in the `logging.debug` call below (as part of `e`) when they have a high enough verbosity set.

Reply:
Changed in my latest push.